FlorenceBML/BMLacq756/BMLacq756.xml

```diff
@@ -29,7 +29,134 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                ወመጽአት፡ ዓሐቲ፡ ብእሲት፡ እንተ፡ ደም፡ ይውሕዛ፡</incipit>
+                            <explicit xml:lang="gez">ወገሰሰት፡ ጽንፈ፡ ለብሱ፡ ወሶ<supplied reason="lost">ቤ</supplied>ሃ፡ ነጸፈ፡ ነቅአ፡ ደማ፡ ከማሃ፡ አሕይዋ፡ ለአመትከ፡ 
+                                <hi rend="rubric"><persName ref="PRS14227WalattaSellase">ወልተ፡ ሥላሴ፡</persName></hi></explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i6">
+                            <locus from="1b1" to="1b159"/>
+                            <title ref="LIT1763Legend"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ 
+                                እ<hi rend="ligature">ግዚ</hi>አ<supplied reason="lost">ብ</supplied>ሔር፡ ሕያው፡</hi>
+                                <gap reason="ellipsis" unit="lines" quantity="2"/> የሐሱ፡ ለቅዱስ፡ <persName ref="PRS8886Sisinios">ሱስንዮስ፡</persName> 
+                                በእንተ፡ አሰሰሎ፡ ዳዌ፡ ወሕፃናት፡ </incipit>
+                            <explicit xml:lang="gez">ወአንስተ፡ ወአንስተ፡ ኢሐውር፡ ውስቴቱ፡ እስከ፡ ለአለመ፡ አለም፡ አሜን፡ ኦቅዱስ፡ 
+                                <persName ref="PRS8886Sisinios">ሱስንዮስ፡</persName>ነ፡ ስማእተ፡ በእ<hi rend="ligature">ግዚ</hi>ኢነ፡ ኢየሱስ፡ ክርስቶስ፡
+                                ሎቱ፡ ስብሃት፡ እስከ፡ ለዓለመ፡ አለም፡ አሜ፡ አድኅኖ፡ ለአመትከ፡ <hi rend="rubric"><persName ref="PRS14227WalattaSellase">ወለተ፡ ስላሴ፡</persName> 
+                                ዘአኧርጋእከ። ነደለ፡ በረ</hi> </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i7">
+                            <locus from="1b160" to="1b189"/>
+                            <title ref="LIT0000000">Prayer against Šatolāy and Šatolāwit</title>
+                            <note>Prayer for healing from vaginal bleeding and for protection of newborn babies through the invocation of magic names
+                                and through the binding of demons with the bonds of the Father, Son and Holy Spirit similar to 
+                                <ref type="work" corresp="LIT6845PPSatolay"/>.</note>
+                            <incipit xml:lang="gez">ጸሎት፡ በእንተ፡ ሸቶላይ፡ ወሸቶላዊት፡ አጋንንት፡ ዘትቀትሎ፡ ሕፃናተ፡ ወታወሕዙ፡ ደመ፡ አመሐኩክሙ፡ ወአውገዝኩከሙ፡ ወአርሰርኩክሙ፡</incipit>
+                            <explicit xml:lang="gez">ደፈር፡ በሎንቃር፡ ፈውስ፡ ወአሕይዋ፡ ለአመትከ፡ 
+                                <hi rend="rubric"><persName ref="PRS14227WalattaSellase">ወልተ፡ ሥላሴ፡</persName></hi>እመ፡ አልቦ፡ ነገር፡ ዘይሰአኖ፡ 
+                                ለእግአብሔር፡</explicit>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Scroll">
+                            <supportDesc>
+                                <support><material key="parchment"/></support>
+                                <extent>
+                                    <measure unit="leaf">1</measure>
+                                    <note>Composed of three parts (670+460+410 mm) that are sewn together with thin leather strips.</note>
```

Comment:
See https://betamasaheft.eu/Guidelines/?id=scrolls for the encoding of the dimensions of each strip